pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+## Description
+
+This PR closes #${issue_number}.
+
+<!-- Write a description to explain changes that are not described in the initatial issues.
+What were the reasons for those changes?
+Which decisions did you make and why? 
+What else should reviewers know about your changes? -->
+
```

Comment:
Good point, I would prefer to add it in the commit body: a lot of things happen and whatever happened it will be in history.